addons/html_editor/static/src/core/clipboard_plugin.js

```diff
@@ -613,12 +645,14 @@ export class ClipboardPlugin extends Plugin {
         if (image || fileTransferItems.length || htmlTransferItem) {
             if (this.document.caretPositionFromPoint) {
                 const range = this.document.caretPositionFromPoint(ev.clientX, ev.clientY);
```

Comment:
You already check for `caretRange` so you can use that here. This `if..else` could be merged